src/sentry_settings.h

```diff
@@ -0,0 +1,43 @@
+#ifndef SENTRY_SETTINGS_H
+#define SENTRY_SETTINGS_H
+
+#include <godot_cpp/core/object.hpp>
+#include <godot_cpp/core/property_info.hpp>
+#include <godot_cpp/variant/char_string.hpp>
+#include <godot_cpp/variant/string.hpp>
+#include <godot_cpp/variant/variant.hpp>
+
+class SentrySettings {
+private:
+	static SentrySettings *singleton;
+
+	bool sentry_enabled = true;
```

Comment:
```suggestion
	bool enabled = true;
```

It's within the `SentryOptions`

Reply:
Addressed in 24cc1ad